code_soup/common/vision/perturbations.py

```diff
@@ -0,0 +1,39 @@
+from abc import abstractmethod
+from typing import Union
+
+import numpy as np
+import torch
+
+from code_soup.common.perturbation import Perturbation
```

Comment:
This file according to the factoring will come under vision-> utils. Please do the needful

Also the common perturbation file should come under common->utils